source/opt/function.h

```diff
@@ -48,17 +48,15 @@ class Function {
   using const_iterator = UptrVectorIterator<BasicBlock, true>;
 
   // Creates a function instance declared by the given instruction |def_inst|.
```

Comment:
We know this is has to be an OpFunction instruction, right?  Might as well say so.